decidim-meetings/app/cells/decidim/meetings/highlighted_meetings_for_component/show.erb

```diff
@@ -1,35 +1,34 @@
 <%# NOTE: Since this call can be called from outside decidim-meetings, frontend assets are required %>
 
-<%= stylesheet_pack_tag "decidim_meetings", media: "all" %>
-<%= javascript_pack_tag "decidim_meetings" %>
+<%= apply_meetings_pack_tags %>
 
-<section id="<%= section_id %>" class="content-block">
-  <div class="content-block__title">
-    <h2 class="h2 decorator">
-      <%= translated_attribute(model.name) %>
-    </h2>
-    <div class="label text-lg"><%= upcoming_meetings? ? upcoming_meetings_count : past_meetings_count %></div>
+<div class="content-block__title">
+  <h2 class="h2 decorator">
+    <%= single_component? ? translated_attribute(model.name) : t("decidim.components.meetings.name") %>
+  </h2>
+  <div class="label text-lg"><%= meetings_count %></div>
+  <% if single_component? %>
     <%= link_to main_component_path(model), class: "button button__sm button__text-secondary" do %>
       <span><%= t("decidim.participatory_spaces.highlighted_meetings.see_all") %></span>
       <%= icon "arrow-right-line" %>
     <% end %>
-  </div>
-  <div class="meeting-list__block-map">
-    <%= cell "decidim/meetings/meetings_map", collection %>
-  </div>
-  <div class="meeting-list__block-list">
-    <% if upcoming_meetings? && false %>
-      <div class="meeting-list__block-list-aside">
-        <% [Date.current, Date.current.next_month].each do |start_date| %>
-          <%= cell "decidim/meetings/meeting_month", collection, start_date: %>
-        <% end %>
-      </div>
-    <% end %>
-    <div class="meeting-list__container">
-      <span class="meeting-list__block-list-title"><%= title %></span>
-      <% collection.includes(:component).each do |meeting| %>
-        <%= cell "decidim/meetings/meeting_l", meeting, link_whole_card: true, title_tag: :h2 %>
+  <% end %>
+</div>
+<div class="meeting-list__block-map">
+  <%= cell "decidim/meetings/meetings_map", collection %>
+</div>
+<div class="meeting-list__block-list">
+  <% if show_calendar? %>
+    <div class="meeting-list__block-list-aside">
+      <% [Date.current, Date.current.next_month].each do |start_date| %>
```

Comment:
Could you move this array of dates to a method in the cell?

Reply:
Done